crates/atuin-macro/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "atuin-macro"
+edition = "2021"
+description = "macro library for atuin"
+
+rust-version = { workspace = true }
+version = "18.5.0-beta.2"
```

Comment:
TODO: this seems to be a cargo-i18n issue that it cannot accept non-literal versions.

That said, running `cargo i18n` doesn't _actually_ seem to be necessary.

Reply:
But might need an upstream change in cargo-i18n to fix :/